python/astro_metadata_translator/observationInfo.py

```diff
@@ -363,7 +443,7 @@ def to_json(self):
         return json.dumps(self.to_simple())
 
     @classmethod
-    def from_simple(cls, simple):
+    def from_simple(cls, simple, extensions=None):
```

Comment:
This parameter is not documented. How does a caller know which extensions are relevant though when they have some serialized JSON? The JSON has the keys but it doesn't know the definitions of the extensions. Maybe we should store the translator class in the JSON (using `lsst.utils.introspection.get_full_type_name`), and by extension in the output of to_simple, so that we can import it (with `lsst.utils.doImportType`) and ask for the extensions?

Does to_simple include the `ext_` in the name so that we know the JSON contains extensions?